MobileCenter/MobileCenter/Internals/Device/MSDeviceTrackerPrivate.h

```diff
@@ -1,8 +1,8 @@
-#if TARGET_OS_IPHONE && !TARGET_OS_TV
+#if TARGET_OS_IOS
 #import <CoreTelephony/CTCarrier.h>
 #import <CoreTelephony/CTTelephonyNetworkInfo.h>
-#import <UIKit/UIKit.h>
-#elif TARGET_OS_TV
+#endif
+#if TARGET_OS_IPHONE
```

Comment:
TargetConditionals.h shows hierarchy of TARGET_OS_* macros.

        TARGET_OS_WIN32           - Generated code will run under 32-bit Windows
        TARGET_OS_UNIX            - Generated code will run under some Unix (not OSX) 
        TARGET_OS_MAC             - Generated code will run under Mac OS X variant
           TARGET_OS_OSX          - Generated code will run under OS X devices
           TARGET_OS_IPHONE          - Generated code for firmware, devices, or simulator
              TARGET_OS_IOS             - Generated code will run under iOS 
              TARGET_OS_TV              - Generated code will run under Apple TV OS
              TARGET_OS_WATCH           - Generated code will run under Apple Watch OS
                 TARGET_OS_BRIDGE          - Generated code will run under Bridge devices
           TARGET_OS_SIMULATOR      - Generated code will run under a simulator
           TARGET_OS_EMBEDDED       - Generated code for firmware
       
        TARGET_IPHONE_SIMULATOR   - DEPRECATED: Same as TARGET_OS_SIMULATOR
        TARGET_OS_NANO            - DEPRECATED: Same as TARGET_OS_WATCH

Yes, TARGET_OS_IPHONE is confusing but it indicates iOS/tvOS/watchOS.
As mentioned offline, we can just do `!TARGET_OS_OSX` but this can also contain `TARGET_OS_SIMULATOR` and `TARGET_OS_EMBEDDED`.